### ISSUE TITLE
[QUEST] You, Robot (10248)

### ISSUE THREAD

**Issue by Kentamentiu** [korakion/Shadowsong#86]
The quest gets marked as complete in the questlog when you take control over the Scrap Reaver, without actually doing the fight.
Negatron, who you are supposed to fight with the Scrap Reaver remains a friendly mob so you can not attack him.
Despite the quest being marked as Complete in the Quest log, it is impossible to turn it in at Doctor Vomisa, Ph.T. He still acts as if the quest is ongoing.
After a few minutes, the  Scrap Reaver becomes hostile and attacks/kills me.

**Comment by Mysterexpo** [comment]
Just tried doing this quest today on my Rogue. Same thing happened to me. The Scrap Reaver doesn't move but I'm able to press the abilities by hand.

**Comment by george352** [comment]
yes, same happened to me. a friendly lv 69 elite standing far there. He suppose to run toward to me so i fight him by controlling the Scrap Reaver.
